### PR TITLE
Adding shift-enter keymap to emulate intellij Start New Line behavior.

### PR DESCRIPTION
**Pull request by jcoulter**
Mapping an additional shortcut.

### CODE REVIEW

File: keymaps/intellij-idea-keymap.cson [hunk 1]
```diff
@@ -230,6 +230,12 @@
   # Duplicate current line or selected block
   'ctrl-d': 'editor:duplicate-lines'
 
+  # Start new line
+  'shift-enter': 'editor:newline-below'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson:234]: What is the difference between `newline` and `newline-below` command? I tested it shortly but I don't see any difference. And `shift-enter` is already bound to `newline` in the default atom keymap.

File: keymaps/intellij-idea-keymap.cson [hunk 2]
```diff
@@ -230,6 +230,12 @@
...
+
+  # Change selected text to upper-case
+  'ctrl-shift-u': 'editor:upper-case'
```
Comment (guylabs) [inline keymaps/intellij-idea-keymap.cson:237]: Could you maybe remove this in favor of the #9 pull request?

File: keymaps/intellij-idea-keymap.cson (hunk 1 above)
Comment (jcoulter) [reply keymaps/intellij-idea-keymap.cson:234]: Open an editor and type text on a new line "1234" 
Place your cursor between "2" and "3" 
With editor:newline, your result is:
"12\n"
"[cursor]34"
With editor:newline-below, your result is:
"1234\n"
"[cursor]" 

I really enjoy this feature.

File: keymaps/intellij-idea-keymap.cson (hunk 2 above)
Comment (jcoulter) [reply keymaps/intellij-idea-keymap.cson:237]: Looks like I messed up my pull request, I intended them to be separate. Sorry, I'll fix it.

**Comment by guylabs** [comment]
Could you maybe add the same shortcut for the mac keybindings as well and remove the one with the upper case? Then I am able to merge it.

Thanks a lot!

Regards,

Gut

**Comment by jcoulter** [comment]
I'll submit a new pull request with the shortcut in the multi-environment section.

**Pull request closed without merging**